checker/jtreg/nullness/issue1275/Main.java

```diff
@@ -0,0 +1,10 @@
+/*
+ * @test
+ * @summary Test case for Issue 1275.
+ * https://github.com/typetools/checker-framework/issues/1275
+ *
+ * @compile -XDrawDiagnostics -processor org.checkerframework.checker.nullness.NullnessChecker#NullnessChecker -AprintErrorStack -Anomsgtext Lib.java
```

Comment:
Remove the '#NullnessChecker' here and in the next line. That's at least one reason for the jtreg failure.